js_modules/dagster-ui/packages/ui-core/src/workers/workerSpawner.ts

```diff
@@ -1,14 +1,35 @@
 import {Worker} from 'shared/workers/Worker.oss';
+
 export function workerSpawner(createWorker: () => Worker) {
-  let nextWorker: Worker = createWorker();
+  function _createWorker() {
+    return createWorker();
+  }
+  function _terminateWorker(worker: Worker) {
+    worker.terminate();
+  }
```

Comment:
I don't think these do anything? I guess it's nice to have consistent naming down below?

Reply:
ah yeah I had some reference counting logic in here before  to debug something... I might add it back though